packages/vue/src/utilities/directives/focus-directive.js

```diff
@@ -0,0 +1,27 @@
+function mousedownHandler(el) {
+  el.style.outline = "";
+}
+
+function keydownHandler(el) {
+  el.blur();
+}
+
+export const focus = {
+  inserted: function(el) {
+    el.tabindex = "0";
+  },
+  bind: function(el) {
+    el.addEventListener("mousedown", function() {
+      el.style.outline = "0";
+      el.blur();
+    });
+    el.addEventListener("keyup", function() {
+      el.style.outline = "";
+      el.focus();
+    });
+  },
+  unbind: function(el) {
+    el.removeEventListener("mousedown", mousedownHandler(el));
+    el.removeEventListener("keyup", keydownHandler(el));
```

Comment:
For consistency
```suggestion
    el.removeEventListener("keydown", keydownHandler(el));
```

Reply:
With the "keydown" in the case when someone focus on click before, it will be impossible to focus on tab - so IMO I think it has to stay that way